templates/students/update_profile.html

```diff
@@ -0,0 +1,150 @@
+                        {% elif profile.gender == 'M' %}
+                            <img class="rounded-circle" style="height: 150px; width: 150px;" src="{% static 'home/images/man.png' %}" alt="profile_image">
+                        {% elif profile.gender == 'F' %}
+                            <img class="rounded-circle" style="height: 150px; width: 150px;" src="{% static 'home/images/woman.png' %}" alt="profile_image">
+                        {% else %}
+                            <img class="rounded-circle" style="height: 150px; width: 150px;" src="{% static 'home/images/man.png' %}" alt="profile_image">
+                        {% endif %}
+                    </div>
+                    <div class="col-md-8 my-3">
+                        <div class="row justify-content-center">
+                            <div class="col-6 col-lg-6 mb-3 align-self-center">
+                                <label for="firstName">First name</label>
+                                <input name="first_name" type="text" class="form-control" id="firstName" value="{{ profile.first_name }}" required>
+                            </div>
+                            <div class="invalid-feedback">
+                                Valid First Name is required.
+                            </div>
+                            <div class="col-6 col-lg-6 mb-3">
+                                <label for="lastName">Last name</label>
+                                <input name="last_name" type="text" class="form-control" id="lastName" value="{{ profile.last_name }}" required>
+                            </div>
+                            <div class="invalid-feedback">
+                                Valid Last Name is required.
+                            </div>
+                        </div>
+                    
+                        <div class="row justify-content-center">
+                            <div class="col-6 col-lg-6 mb-3 align-self-center">
+                                <label for="gender">Gender</label>
+                                <select name="gender" class="custom-select d-block w-100" id="gender" required>
+                                    <option value="M" {% if  profile.gender == "M" %} selected {% endif %} >Male</option>
+                                    <option value="F" {% if  profile.gender == "F" %} selected {% endif %} >Female</option>
+                                    <option value="O" {% if  profile.gender == "O" %} selected {% endif %} >Other</option>
+                                </select>
+                            </div>
+                            <div class="col-6 col-lg-6 mb-3">
+                                <label for="schoolClass">Class</label>
+                                <input name="school_class" type="number" class="form-control" id="school_class" value="{{ profile.school_class }}" min="0" max="12" required>
```

Comment:
input id should match with the for attribute of label.

Also, keep `min` as 1 and `max` as 15 (we keep 13 class for pass-out students and 14, 15 for Navodaya students).